resources/dicts/events/lifegen_events/murder.json

```diff
@@ -284,8 +284,9 @@
         "Lately v_c has been complaining and meowing on about how their always in pain and no matter how many poppy seeds you give them a mixture of deathberries, foxglove seeds, a single leaf of catmint and poppy seeds, one herb to hide the foul bitter taste of the poisonous herbs and another herb to put them to sleep before they can realize what happened, v_c is later found dead in their nest in the elders den."
     ],
     "murder mediator kitten": [
-        "You encourage v_c to eat some shiny red berries you've found, saying that they're super tasty - better than freshkill even! Happily, they thank you for the snack. v_c is later found, lifeless, with deathberry juice dripping from their mouth.",
-        "You don't know who that cat you saw in your dream was, but they gave you one clear goal. The memories of that shadowy, foggy forest flash in your mind as you sneak v_c out of camp before any cat wakes up. You have time to return and clean up before anyone even awakens, and when the Clan realizes what happened, they only conclude it must've been some injury or predator attack from sneaking out."
+        "You encourage v_c to eat some shiny red berries you've found, saying that they're super tasty - better than freshkill even! Happily, they thank you for the snack. v_c is later found, lifeless, already having gone to StarClan.",
```

Comment:
this can be changed to match the previous one

"You encourage v_c to eat some shiny red berries you've found, and after eating them, v_c is later found having travelled to meet Silverpelt early."